### ISSUE TITLE
[XCUI Screenshot Test]  test4PasscodeSettings fails when language is arabic

### ISSUE THREAD

**Issue by isabelrios** [mozilla-mobile/firefox-ios#6587]
## Steps to reproduce
Set the device in Arabic 'ar' and run the [test](https://github.com/mozilla-mobile/firefox-ios/blob/master/L10nSnapshotTests/L10nSuite1SnapshotTests.swift#L73)

### Expected behavior
Test successes 

### Actual behavior
Test fails trying to type the passcode '111111'
![Screenshot 2020-05-11 at 17 44 18](https://user-images.githubusercontent.com/1897507/81582032-d4626700-93af-11ea-92ed-344474c3bbd2.png)

This is the function to do that: https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L805

If we tap on  '123' key, before, then the test  work. A workaround would be to  set  a  condition in the test if the app is in arabic to tap that key before.

**Comment by isabelrios** [comment]
The nice way should be to use: 
```
if (UIApplication.shared.userInterfaceLayoutDirection == .rightToLeft) {
    app.keys["123"].tap()
}
```
But that never returns true, the Layaout direction  does not seem to be taken into account. 
Another option would be:
```
if (app.keys["123"].exists) {
     app.keys["123"].tap()
 }
```

**Comment by isabelrios** [comment]
Unfortunately this is still happening when running the tests

**Comment by isabelrios** [comment]
Closed by #6600
